### PR TITLE
Headless mode execution

### PR DESCRIPTION
**Pull request by nebgnahz**
Only supported on Linux right now.

### CODE REVIEW

File: Xcode/ESP/src/ofApp.cpp [hunk 1]
```diff
@@ -502,7 +511,9 @@ void ofApp::setup() {
     // Register myself as logging observer but disable first.
     GRT::ErrorLog::enableLogging(false);
     GRT::ErrorLog::registerObserver(*this);
-}
+
+    loadAll(save_path_);
```
Comment (damellis) [inline Xcode/ESP/src/ofApp.cpp]: Does this cause problems is save_path_ isn't set?

**Comment by nebgnahz** [comment]
It should be checked (fixed already).

Right now it doesn't cause issue  because each individual loading function will return boolean value for loading results; and if any fails, the overall loading fail. But this doesn't cause the program to crash.

See: https://github.com/damellis/ESP/blob/6009aa0b28595ccf9332bdd3a69f357158d507f4/Xcode/ESP/src/ofApp.cpp#L1026

File: Xcode/ESP/src/ofApp.cpp (hunk 1 above)
Comment (nebgnahz) [inline Xcode/ESP/src/ofApp.cpp]: Fixed in 60edc60

**Comment by nebgnahz** [comment]
This PR has been obsolete. Closing for now.

**Pull request closed without merging**